_inc/client/at-a-glance/plugins.jsx

```diff
@@ -26,6 +27,20 @@ class DashPluginUpdates extends Component {
 		pluginUpdates: PropTypes.any.isRequired,
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.trackManagePlugins = this.trackManagePlugins.bind( this );
```

Comment:
We don't really need this binding as `trackManagePlugins` makes no reference to `this` in its code.